### PR TITLE
Refactoring, Coding Standard, and Fomatting done

### PR DESCRIPTION
**Pull request by aj3sh**
- Added CONTRIBUTION.md
- Bug fix for django templatetags
- None checking update with PEP8 standard
- nepali.utils updated
- Code formatted
- Removed black check only params on pre-commit hooks
- Added unittest for utils and timezone codes

### CODE REVIEW

File: nepali/datetime/utils.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+def to_nepalidatetime(datetime_object: Any) -> nepalidatetime:
+    """
+    Converts nepalidate, datetime.datetime, datetime.date to nepalidatetime.
+
+    :param datetime_object: Object to be converted into nepalidatetime
+    :return: nepalidatetime
+    :raises InvalidNepaliDateTimeObjectException: If the input data is not a date time objects
+    """
+    if type(datetime_object) == nepalidatetime:
```
Comment (sugat009) [inline nepali/datetime/utils.py]: lets go with `isinstance(datetime_object, nepalidatetime)` as it's more pythonic

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 coverage
 pytz
-black==22.10.0
-pre-commit==3.0.4
+black==23.1.0
+pre-commit==2.21.0
```
Comment (sugat009) [inline requirements.txt]: any particular reasons for the downgrade?

**Review by sugat009 — CHANGES_REQUESTED** [review]

File: CONTRIBUTION.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# How to contribute
```
Comment (sugat009) [inline CONTRIBUTION.md:1]: Lets mention this file in readme.md as well

**Review by sugat009 — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (aj3sh) [reply requirements.txt]: That was a mistake during the upgrading process.

File: CONTRIBUTION.md (hunk 3 above)
Comment (aj3sh) [reply CONTRIBUTION.md:1]: Done

File: nepali/datetime/utils.py (hunk 1 above)
Comment (aj3sh) [reply nepali/datetime/utils.py]: Done

Comment (aj3sh) [reply nepali/datetime/utils.py]: Test cases added to verify this

**Review by sugat009 — APPROVED** [review]

**Pull request merged**